packages/core/src/types.ts

```diff
@@ -20,13 +20,19 @@ export type ConnectConfig = {
 };
 
 export type SocialConnectMethod = "google" | "telegram" | "discord" | "apple";
+export type SocialConnectMethodMap = { [K in SocialConnectMethod]: true };
 
 export type ConnectMethod =
   | SocialConnectMethod
   | "email"
   | "passkey"
   | "wallet";
 
+export function isSocialConnectMethod(method: ConnectMethod): method is SocialConnectMethod {
```

Comment:
I want to keep this file to exporting types only. Mind moving this to a util file somewhere?